### DISCUSSION TITLE
Does opentelemetry java autoinstrumentation support frames

### DISCUSSION THREAD

**Discussion by satwika007** [open-telemetry/opentelemetry-java-instrumentation#3754] [Q&A]
I'm trying to autoinstrument my java application which has frames. I want to know if javaagent can take care of frames or should we add any code manually?

**Comment by trask** [comment]
hi @satwika007! can you clarify you question? I'm not sure if you mean stack map frames, or maybe something else?

**Reply by jkwatson** [reply]
@trask would it make sense to move this discussion over to the instrumentation repo?

**Reply by trask** [reply]
> @trask would it make sense to move this discussion over to the instrumentation repo?

yes, please
